### PR TITLE
Correct project dependencies

### PR DESCRIPTION
**Pull request by dougbu**
- use latest `System.Net.Http`, not hard-coded version
- add imports for the latest CLI

### CODE REVIEW

**Comment by dougbu** [comment]
/cc @NTaylorMullen @victorhurdugaci 

This gets the repo building locally w/o warnings.

**Comment by victorhurdugaci** [comment]
Looks good to me. Thanks for fixing what I broke :)

:shipit:

**Comment by dougbu** [comment]
7c7a4a9

File: test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/project.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "dependencies": {
     "AntiforgerySample": "1.0.0-*",
     "Microsoft.AspNetCore.TestHost": "1.0.0-*",
-    "Microsoft.AspNetCore.Testing": "1.0.0-*"
+    "Microsoft.AspNetCore.Testing": "1.0.0-*",
+    "Microsoft.NETCore.Platforms": "1.0.1-*"
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/project.json]: This gets brought in transitively.

File: test/Microsoft.AspNetCore.Antiforgery.Test/project.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "Microsoft.AspNetCore.Http": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
+    "Microsoft.NETCore.Platforms": "1.0.1-*",
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Antiforgery.Test/project.json]: Talked with @davidfowl about when we should apply the netcore platforms dependency and we only need to do it when there's external dependencys included in the project (newtonsoft,autofac, things like that)

Comment (dougbu) [inline test/Microsoft.AspNetCore.Antiforgery.Test/project.json]: `xunit`

File: test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/project.json (hunk 1 above)
Comment (dougbu) [inline test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/project.json]: This code is checked in. Worth another commit or wait for next time we change this file?

Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/project.json]: If you feel inspired go for it. Not a big deal though :smile:

**Pull request closed without merging**